libnetwork_test.go

```diff
@@ -251,8 +251,6 @@ func TestBridge(t *testing.T) {
 			"FixedCIDR":             cidr,
 			"FixedCIDRv6":           cidrv6,
 			"EnableIPv6":            true,
-			"EnableIPTables":        true,
-			"EnableIPMasquerade":    true,
```

Comment:
if not necessary, let's not remove the enable ip masquerade from the network config in this test. This specific case we are trying to pass all the possible settings.